honeybee_radiance/writer.py

```diff
@@ -238,141 +239,79 @@ def model_to_rad_folder(model, folder=None, minimal=False):
-                        nonopaque_shades, nonopaque_shades_blk, nos_mods, nos_mods_blk,
-                        mod_combs, mod_names, False, minimal)
-
-    # gather static outdoor shades
-    opaque_shades, opaque_shades_blk, nonopaque_shades, nonopaque_shades_blk = \
-        model.properties.radiance.outdoor_shades_by_opaque()
-
-    # write opaque static outdoor shades
-    os_mods, os_mods_blk, mod_combs, mod_names = \
-        _collect_modifiers(opaque_shades, opaque_shades_blk, True)
-    _write_static_files(folder, model_folder.STATIC_OPAQUE_OUTDOOR, model_id,
-                        opaque_shades, opaque_shades_blk, os_mods, os_mods_blk,
-                        mod_combs, mod_names, False, minimal)
-
-    # write nonopaque static outdoor shades
-    nos_mods, nos_mods_blk, mod_combs, mod_names = \
-        _collect_modifiers(nonopaque_shades, nonopaque_shades_blk, False)
-    _write_static_files(folder, model_folder.STATIC_NONOPAQUE_OUTDOOR, model_id,
-                        nonopaque_shades, nonopaque_shades_blk, nos_mods, nos_mods_blk,
-                        mod_combs, mod_names, False, minimal)
+    model_folder.write(folder_type=folder_type, cfg=folder_config.full, overwrite=True)
+
+    # gather and write static apertures to the folder
+    aps, aps_blk = model.properties.radiance.subfaces_by_blk()
+    mods, mods_blk, mod_combs, mod_names = _collect_modifiers(aps, aps_blk)
+    _write_static_files(
+        folder, model_folder.aperture_folder(full=True), model_id,
+        aps, aps_blk, mods, mods_blk, mod_combs, mod_names, False, minimal)
+
+    # gather and write static faces
+    faces, faces_blk = model.properties.radiance.faces_by_blk()
+    f_mods, f_mods_blk, mod_combs, mod_names = _collect_modifiers(faces, faces_blk)
+    _write_static_files(
+        folder, model_folder.scene_folder(full=True), '{}_envelope'.format(model_id),
+        faces, faces_blk, f_mods, f_mods_blk, mod_combs, mod_names, True, minimal)
+
+    # gather and write static shades
+    shades, shades_blk = model.properties.radiance.shades_by_blk()
+    s_mods, s_mods_blk, mod_combs, mod_names = _collect_modifiers(shades, shades_blk)
+    _write_static_files(
+        folder, model_folder.scene_folder(full=True), '{}_shades'.format(model_id),
+        shades, shades_blk, s_mods, s_mods_blk, mod_combs, mod_names, False, minimal)
 
     # write dynamic sub-face groups (apertures and doors)
     ext_dict = {}
+    out_subfolder = model_folder.aperture_group_folder(full=True)
     for group in model.properties.radiance.dynamic_subface_groups:
         if group.is_indoor:
             # TODO: Implement dynamic interior apertures once the radiance folder
             # structure is clear about how the "light path" should be input
             raise NotImplementedError('Dynamic interior apertures are not currently'
                                       ' supported by Model.to.rad_folder.')
         else:
```

Comment:
We can now support the indoor aperture group. You can just copy them to aperture_group_folder(full=True, indoor=True). With the addition of light-path to grids now we know how to treat them for simulations.

As for the direction that I know you asked about before I think we should have two and name them based on the rooms/faces that share this interior aperture. It will always be facing inwards towards that room similar to how it is currently written for the exterior aperture group.

Reply:
Ah, I didn't realize that. So the fact that we have the `light_path` that with the grid files means that we don't need to include this information within the interior apertures folder?  If so, that helps a lot.

A sample of what you mean by "both directions" would be really helpful. Does this mean that both directions go within the same .rad file? Or if they are separate, how are they differentiated from one another?

I agree that it's better to merge this now and, if you can add a sample or at least a better description to [this part of the radiance-folder-structure](https://github.com/ladybug-tools/radiance-folder-structure/tree/master/radiance_folder/model/aperture_group/interior), I will implement what you document.